src/services/namex-services.ts

```diff
@@ -141,9 +141,8 @@ export default class NamexServices {
    */
   static async searchColin (corpNum: string): Promise<any> {
     // remove BC prefix as COLIN only supports base number with no prefix for BC's
-    const cleanedCorpNum = corpNum.replace(/^BC+/i, '')
-    const url = `${appBaseURL}/colin/${cleanedCorpNum}`
-    return this.axios.post(url).then(response => response.data)
+    const url = `${appBaseURL}/colin/${corpNum}`
+    return this.axios.get(url).then(response => response.data)
```

Comment:
Awesome! But shouldn't this be a new Namex API endpoint?

Reply:
this should be the new endpoint https://github.com/bcgov/namex/pull/1413#discussion_r1321763306

Reply:
It looks like the same endpoint to me...?

Reply:
existing endpoint is POST and the new one is GET

Reply:
Ah, OK, good.

Would it make sense to call it something other than "COLIN"?

Reply:
I think colin make more sense since we are searching in colin db

Reply:
OK, leave it as is then.

(We search both LEAR and COLIN, right?)

Reply:
yes